cartographer/mapping/pose_graph_data.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 The Cartographer Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CARTOGRAPHER_MAPPING_POSE_GRAPH_DATA_H
+#define CARTOGRAPHER_MAPPING_POSE_GRAPH_DATA_H
+
+#include "cartographer/mapping_2d/pose_graph.h"
+#include "cartographer/mapping_3d/pose_graph.h"
+#include "cartographer/sensor/data.h"
+
+namespace cartographer {
+namespace mapping {
+
+class TrajectoryBuilderInterface;
+class PoseGraphData : public cartographer::sensor::Data {
```

Comment:
rename to LocalSlamResultData (and update the file names)

Reply:
Done.